webrender/src/renderer.rs

```diff
@@ -664,7 +658,7 @@ impl Renderer {
             profile_counters: RendererProfileCounters::new(),
             profiler: Profiler::new(),
             enable_profiler: options.enable_profiler,
-            enable_msaa: options.enable_msaa,
+            _enable_msaa: options.enable_msaa,
```

Comment:
Ok, I removed some of the MSAA logic, but left the `RenderOptions` (to avoid breaking the API, who knows if we'll need it back soon?) as well as the device's capability field.